targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/TARGET_MCU_NRF52840/CMakeLists.txt

```diff
@@ -1,20 +1,65 @@
 # Copyright (c) 2020 ARM Limited. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-if("NRF52840_DK" IN_LIST MBED_TARGET_LABELS)
-    add_subdirectory(TARGET_NRF52840_DK)
+if ("ARDUINO_NANO33BLE" IN_LIST MBED_TARGET_LABELS)
+    target_include_directories(mbed-core
+        INTERFACE
+            TARGET_ARDUINO_NANO33BLE
+    )
+elseif("EP_AGORA" IN_LIST MBED_TARGET_LABELS)
+    target_include_directories(mbed-core
+        INTERFACE
+            TARGET_EP_AGORA
+    )
+    target_sources(mbed-core
+        INTERFACE
+            TARGET_EP_AGORA/ONBOARD_TELIT_ME910.cpp
+    )
+elseif("EP_ATLAS" IN_LIST MBED_TARGET_LABELS)
+    target_include_directories(mbed-core
+        INTERFACE
+            TARGET_EP_ATLAS
+    )
+# TODO CMAKE this file TARGET_EP_ATLAS/ONBOARD_TELIT_ME310.cpp should probably
```

Comment:
@ARMmbed/team-nordic what is your recommendation for this?

Reply:
we should move these files to cellular drivers as proposed.

Reply:
@ARMmbed/team-embeddedplanet ^^

Reply:
Lets create new issue and we can continue with this one as it is.